ui/types/extras/icon-set.d.ts

```diff
@@ -0,0 +1,92 @@
+import { StringDictionary } from "../ts-helpers";
+
+type QuasarIconSetType = "positive" | "negative" | "info" | "warning";
+type QuasarIconSetArrow = "up" | "right" | "down" | "left" | "dropdown";
+type QuasarIconSetChevron = "left" | "right";
+type QuasarIconSetColorPicker = "spectrum" | "tune" | "palette";
+type QuasarIconSetPullToRefresh = "icon";
+type QuasarIconSetCarousel = "left" | "right" | "navigationIcon" | "thumbnails";
+type QuasarIconSetChip = "remove" | "selected";
+type QuasarIconSetDatetime = "arrowLeft" | "arrowRight" | "now" | "today";
+type QuasarIconSetEditor =
+  | "bold"
```

Comment:
Is the `|` before the first option going to be a problem?  It should probably be removed.

Reply:
No, it's only a style choice and it will be ignored during TS parsing. Prettier uses this notation for readability, keeping all options aligned